src/ws/cockpitwebservice.c

```diff
@@ -457,11 +457,6 @@ process_socket_authorize (CockpitWebService *self,
   cockpit_memory_clear (data, length);
 
 out:
-  if (password)
-    {
-      cockpit_memory_clear (password, -1);
-      free (password);
-    }
```

Comment:
Only place this is really used is in the parse from basic part. So I makes sense to either remove this or replace the clearing there with goto out.